tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright © 2023 - 2024 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import 'cypress/support/commands';
+import * as cypressLib from '@rancher-ecp-qa/cypress-library';
+import { qase } from 'cypress-qase-reporter/dist/mocha';
+
+beforeEach(() => {
+  cy.login();
+  cy.visit('/');
+});
+
+Cypress.config();
+describe('Test Fleet access with RBAC with custom roles', { tags: '@rbac' }, () => {
+
+  const baseUser      = "base-user"
+  const uiPassword    = "rancherpassword"
+
+  qase(5,
+    it('Test "User-Base" user with custom role to "fleetworkspaces", "gitrepos" and "bundles" and  ALL verbs access can access "Workspaces", "Bundles" and "Git Repos" but not to "Clusters" and "Clusters Groups"', { tags: '@fleet-5' }, () => {
+
+      // Create User "User-Base"
+      cypressLib.burgerMenuToggle();
+      cypressLib.createUser(baseUser, uiPassword, "User-Base", true);
+
+      // Create role
+      cy.createRoleTemplate({
+        roleType: "Global",
+        roleName: "fleetAllVerbsRole",
+        newUserDefault: "Yes",
+        resources: ["fleetworkspaces", "gitrepos", "bundles"],
+        verbs: ["create", "delete", "get", "list", "patch", "update", "watch"],
+        apiGroups: ["*"],
+        nonResourcesURLs: ["*"]
+      });
+
+      // Assign role ti created user
+      cy.assignRoleToUser(baseUser, "fleetAllVerbsRole");
+      
+      // Logout as admin and login as other user
+      cypressLib.logout();
+      cy.login(baseUser, uiPassword);
+      
+      // What the user should be able to access
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
```

Comment:
After navigating to this and below menu's, assert Titles available on the page to ensure it lands on the navigation.

Reply:
I introduced for [this](https://github.com/rancher/fleet-e2e/pull/128/files#diff-75965941f29be613c2bab5bdb432b711b86c52a2567d7f08dcf8aaeef907eb0cR157) pr an assertion based on visibility of accessed term both for first and second access menu:
`cy.contains(firstAccessMenu).should('be.visible')`

If we do it based on title (ie, using `cy.contains('div[class="outlet"] div[class="title"]', firstAccessMenu).should('be.visible')) `, when the title does not match like with `Clusters` whose title is `Cluster Dashboard` or exist, like in `Home` which has nothing, the assertion will fail.

If you find an easy way, please share, otherwise I will leave as it is for this pr.

Reply:
Okay, I fine with current implementation. I missed that part while reviewing it. Thanks :smile: